src/lists.cpp

```diff
@@ -202,6 +202,21 @@ namespace {
 
 // ============================================================================================================
 
+namespace
+{
+   std::map<std::string_view, std::pair<std::string_view, std::string_view>, std::less<>> substitutions {
+      { "discussion", {"<p><b>Discussion:</b></p>", ""} },
+         { "resolution", {} },
+         { "rationale", {"<p><b>Rationale:</b></p>", ""} },
+         { "duplicate", {} },
+         { "note", { "<p><i>[", "]</i></p>\n"} },
+         { "superseded",
```

Comment:
Could we also add `<!ELEMENT superseded ANY>` to the DTD to keep it current?

Reply:
Oh yes, I forgot about the DTD. Done, thanks!